packages/core/src/components/tabs/tab.tsx

```diff
@@ -61,6 +64,18 @@ export interface ITabProps extends Props, Omit<HTMLDivProps, "id" | "title" | "o
      * Can also be set via React `children`.
      */
     title?: React.ReactNode;
+
+    /** Name of a Blueprint UI icon (or an icon element) to render before the children. */
+    icon?: IconName | MaybeElement;
+
+    /** Content to render inside a `<Tag>` after the children. */
+    tagContent?: TagProps["children"];
+
+    /**
+     * Props to customize the `<Tag>` rendered after the children.
```

Comment:
we should note that these tags are minimal by default, so `{ minimal: true }` is unnecessary